spec/models/concerns/devise_overrides_spec.rb

```diff
@@ -0,0 +1,174 @@
+require 'rails_helper'
+
+describe DeviseOverrides do
+  let(:extended_class) { Class.new { extend ActiveSupport::Concern } }
+  
+  describe '.confirmed?' do
+    let(:customer) { Customer.new(password: '123451', password_confirmation: '123451', email: 'test@gmail.com') }
+    it 'return confirmed status' do
+      expect(customer.confirmed?).to eq(customer.active?)
+    end
+  end
+  
+  describe '.confirmation_required?' do
```

Comment:
Do we use this method `confirmation_required?`? why it returns `false` all the time?